contracts/script/GenerateAlloc.s.sol

```diff
@@ -217,25 +197,34 @@ contract GenerateAlloc is Script {
         // We deploy this with Create3 because we can't set storage variables in constructor with vm.etch
 
         uint256 minDelay = getTimelockMinDelay();
-        (address[] memory proposers, address[] memory executors, address canceller) = getTimelockControllers();
+        address[] memory proposers = new address[](1);
+        proposers[0] = protocolAdmin;
+        address[] memory executors = new address[](1);
+        executors[0] = timelockExecutor;
+        address canceller = timelockGuardian;
```

Comment:
I guess this hasn't been changed to multi-sig yet?

Reply:
No need to change anything here, multisigs are identified by EVM address, just need to have the right .env values